lib/webdriver.py

```diff
@@ -139,8 +139,9 @@ def _get_driver(self) -> Driver:
             driver_version=driver_version,
             headless=True,
             uc_cdp_events=True,
-            undetectable=True,
+            uc=True,
```

Comment:
In SeleniumBase, `uc` [is an alias of](https://github.com/seleniumbase/SeleniumBase/blob/af3d9545473e55b2a25cdbab8be0b1ed5e1f6afa/seleniumbase/plugins/driver_manager.py#L123) `undetectable`. We should leave it as `undetectable` to make the Driver parameters more explicit and easier to read.